### ISSUE TITLE
Deleting a SSO-generated user from Jellyfin prevents account from working

### ISSUE THREAD

**Issue by jmuchovej** [9p4/jellyfin-plugin-sso#98]
**Describe the bug**
After creating a user using SSO, if that user is deleted, they can no longer login (even if they still have permissive access – i.e., are still allowed to access Jellyfin per roles).

**To Reproduce**

1. Setup Jellyfin SSO in the usual way.
2. Login using an SSO account.
3. As the Jellyfin admin (non-SSO account), delete the newly added SSO user.
4. Attempt to sign-in using the same SSO account.

**Expected behavior**
Their account would be reinstated or a new one would be created with the proper attributes.

**Additional context**
Much of the [discussion starting at my first post][post] in #96 could be relevant.

[post]: https://github.com/9p4/jellyfin-plugin-sso/issues/96#issuecomment-1304623390

**Comment by basbebe** [comment]
To add in here:
For me, those users will still be able to login via LDAP.
A new account will be generated but those users will **not** be able to login via SSO anymore.

**Comment by basbebe** [comment]
Is there a way to 'reset' those connections without purging / reinstalling Jellyfin?

**Comment by 9p4** [comment]
Yes. If you stop Jellyfin, edit the plugin configuration file `config/plugins/configurations/SSO-Auth.xml` and remove the item from `CanonicalLinks` that you want to reset and start the Jellyfin server again, that should remove the link.

**Comment by basbebe** [comment]
I deleted **all** entries within `CanonicalLinks` which resulted in not being to login as any user via SSO and the provider had disappeared from the settings.
I reentered the provider and now everything works again.

**Comment by xNinjaKittyx** [comment]
If anyone runs into this again, when you remove CanonicalLinks and you don't have any entries left, make sure it's not
```
<CanonicalLinks>
</CanonicalLinks>
```

and instead you use
```
<CanonicalLinks />
```

Otherwise your entire configuration is going to get wiped and you'll have to fill it out again.
